tests/full_wlm/test_wlm_helper_functions.py

```diff
@@ -8,6 +8,8 @@
 
 
```

Comment:
Would you mind adding a short comment here explaining why alloc_specs might not be defined? E.g.,

```
# Slurm or PBS system can sometimes be set up such that not all 
#   environment variables associated with 'alloc_specs' are populated,
#   so 'alloc_specs'-related tests only run when the environment variables are set
```

Reply:
I'll add it to the tests. This feature is still under-documented, anyhow.